### DISCUSSION TITLE
What's the color theme used in the landing page hero image?

### DISCUSSION THREAD

**Discussion by ffpetrovic** [codesandbox/codesandbox-client#4603] [General] [answered]
What's the color theme used in the landing page hero image?

![](https://codesandbox.io/static/hero-ide-home-4ca67e8548248798ba899468ba3fa64e.png)

**Comment by MichaelDeBoey** [comment]
@DannyRuchtie will be the best guy to answer this I guess, but I think it's not a particular theme, it's just done by hand I think.

**Comment by ffpetrovic** [comment]
Seems like it to me as well, especially the code map on the right side.

Thanks anyway :)

**Comment by DannyRuchtie** [comment, marked as answer]
Hi @ffpetrovic 
It's from a theme I'm currently working on (the screenshot is done by hand in Figma :) )
We did not add it to the theme yet, but you can find the Prism theme over here (WIP):
https://codesandbox.io/s/ide-dashboard-2lmeh?file=/src/theme/dark.js
